src/3d/qgscameracontroller.cpp

```diff
@@ -741,6 +881,50 @@ void QgsCameraController::onKeyPressedTerrainNavigation( Qt3DInput::QKeyEvent *e
   }
 }
 
+void QgsCameraController::onKeyPressedGlobeTerrainNavigation( Qt3DInput::QKeyEvent *event )
+{
+  const float moveFactor = 0.000001f; // multiplied by distance to get angle
+  const float zoomFactor = 0.9f;
+
+  const bool hasShift = ( event->modifiers() & Qt::ShiftModifier );
+
```

Comment:
How about ctrl modifier = 1/10 movement factor, like we use elsewhere?

Reply:
yeah, that should come later, with a couple of other improvements to bring it on par with the current terrain-based navigation (e.g. zooming/rotation with pivot point away from window center) - but I'd like to keep this PR minimal in scope